vignettes/ebola.Rmd

```diff
@@ -0,0 +1,615 @@
+
+```{r linpred-sex-district, fig.cap="(ref:linpred-sex-district)"}
+linpred_draws_sex_district |>
+  dplyr::mutate(
+    sex = dplyr::case_when(
+      sex == "F" ~ "Female",
+      sex == "M" ~ "Male"
+    )
+  ) |>
+  ggplot(aes(x = mu, y = district)) +
+  tidybayes::stat_pointinterval() +
+  facet_grid(. ~ sex) +
+  labs(x = "Posterior of the mu linear predictor", y = "") +
+  theme_minimal()
+```
+
+## Delay probability mass functions
+
+Posterior predictions of the delay distribution are an important output of an analysis with the `epidist` package.
+In this section, we demonstrate how to produce either a discrete probability mass function representation, or continuous probability density function representation of the delay distribution.
+
+### Discrete probability mass function
+
+The posterior predictive distribution under no truncation but daily censoring.
+
+No stratification:
+
+```{r}
+newdata_pmf <- obs_prep |>
+  as.data.frame() |>
+  dplyr::mutate(obs_t = 1000, pwindow_upr = 1, swindow_upr = 1)
+
+draws_pmf <- predicted_draws(fit, newdata = newdata_pmf, ndraws = 1000)
+
+ggplot(draws_pmf, aes(x = .prediction)) +
+  geom_bar(aes(y = after_stat(count / sum(count)))) +
+  labs(x = "PMF with daily censoring and no truncation", y = "") +
+  theme_minimal()
+```
+
+Stratification by sex:
+
+```{r}
+newdata_sex_pmf <- obs_prep |>
+  as.data.frame() |>
+  modelr::data_grid(sex) |>
+  dplyr::mutate(obs_t = 1000, pwindow_upr = 1, swindow_upr = 1)
+
+draws_sex_pmf <- predicted_draws(
+  fit_sex,
+  newdata = newdata_sex_pmf,
+  ndraws = 1000
+)
+
+draws_sex_pmf |>
+  dplyr::mutate(
+    sex = dplyr::case_when(
+      sex == "F" ~ "Female",
+      sex == "M" ~ "Male"
+    )
+  ) |>
+  ggplot(aes(x = .prediction)) +
+  geom_bar(aes(y = after_stat(count / sum(count)))) +
+  labs(x = "PMF with daily censoring and no truncation", y = "") +
+  facet_grid(. ~ sex) +
+  theme_minimal()
+```
+
+Stratification by sex-district:
+
+```{r}
+newdata_sex_district_pmf <- obs_prep |>
+  as.data.frame() |>
+  modelr::data_grid(sex, district) |>
+  dplyr::mutate(obs_t = 1000, pwindow_upr = 1, swindow_upr = 1)
+
+draws_sex_district_pmf <- predicted_draws(
+  fit_sex_district,
+  newdata = newdata_sex_district_pmf,
+  ndraws = 1000
+)
+
+draws_sex_district_pmf |>
+  dplyr::mutate(
+    sex = dplyr::case_when(
+      sex == "F" ~ "Female",
+      sex == "M" ~ "Male"
+    )
+  ) |>
+  ggplot(aes(x = .prediction)) +
+  geom_bar(aes(y = after_stat(count / sum(count)))) +
+  labs(x = "PMF with daily censoring and no truncation", y = "") +
+  facet_grid(district ~ sex) +
+  theme_minimal()
+```
+
+### Continuous probability density function
+
+The posterior predictive distribution under no truncation and no censoring.
+That is to produce continuous delay times.
+I don't like that `geom_density()` is doing kernel smoothing but plotting as a histogram is suggestive of there being discrete structure.
```

Comment:
remove this as the user doesn't need to know

Reply:
Done